src/modules/help.ts

```diff
@@ -0,0 +1,85 @@
+import { command, default as CookiecordClient, Module } from 'cookiecord';
```

Comment:
Run `yarn tsc` locally and fix the errors. (GitHub actions seems to have messed up the line numbers)